packages/shared/src/broker.ts

```diff
@@ -134,7 +148,11 @@ const makeRpcRequest = async <T extends keyof RequestValidator & keyof ResponseV
 
 export async function requestSwapDepositAddress(
   swapRequest: NewSwapRequest,
-  opts: { url: string; commissionBps: number },
+  opts: {
+    url: string;
+    commissionBps: number;
+    affiliateBrokers?: AffiliateBroker[];
```

Comment:
we dont have a `broker` prefix for `commissionBps`, so we can rename this to `affiliates`